### PR TITLE
[15.0][IMP] web_refresher: Provide the ability to reload information from a report view

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT47565

ping @sergio-teruel @carlosdauden

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Cool :+1:

File: web_refresher/static/src/js/refresher.esm.js [hunk 1]
```diff
@@ -1,15 +1,28 @@
 /** @odoo-module **/
 /* Copyright 2022 Tecnativa - Alexandre D. Díaz
- * Copyright 2022 Tecnativa - Carlos Roca
+ * Copyright 2024 Tecnativa - Carlos Roca
```
Comment (carlosdauden) [inline web_refresher/static/src/js/refresher.esm.js]:
```suggestion
 * Copyright 2022-2024 Tecnativa - Carlos Roca
```

**Review by carlosdauden — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
Change done @carlosdauden

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please fw-port it to v16:

/ocabot merge patch

**Pull request merged**